packages/agent-cli/src/copilot/index.ts

```diff
@@ -0,0 +1,118 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+function findAllNodeModulesDirectories(
+	directory: string,
+	found: string[] = []
+): string[] {
+	if (!fs.existsSync(directory)) {
+		return found;
+	}
+
+	const entries = fs
+      .readdirSync(directory, { withFileTypes: true })
+      .sort((a, b) => a.name.localeCompare(b.name, "en"));
+	for (const entry of entries) {
+		if (entry.isDirectory()) {
+			if (entry.name === 'node_modules') {
+				found.push(path.join(directory, entry.name));
+			} else if (!entry.name.startsWith('.')) {
+				findAllNodeModulesDirectories(
+					path.join(directory, entry.name),
+					found
+				);
+			}
+		}
+	}
+
+	return found;
+}
+
+export const generateCopilot = (rootPath: string) => {
+	const outputFolder = path.resolve(rootPath, '.github');
+
+	const nodeModulesDirectories = findAllNodeModulesDirectories(rootPath);
+	if (nodeModulesDirectories.length === 0) {
+		console.error('No node_modules folders found.');
+		return;
+	}
+
+	let copilotInstructionsContent = '';
+
+	for (const nodeModulesPath of nodeModulesDirectories) {
+		const databaseUxPaths = [
+			path.join(nodeModulesPath, '@db-ux'),
+			path.join(nodeModulesPath, '@db-ux-inner-source')
```

Comment:
```suggestion
			path.join(nodeModulesPath, '@db-ux/'),
			path.join(nodeModulesPath, '@db-ux-inner-source/')
```

this would be clearer that it's a package out of our npmjs.com organisation.